heidegger_index/md.py

```diff
@@ -63,13 +54,48 @@ def handleMatch(self, m, data):
         else:
             href = reverse("work-detail", kwargs={"slug": work_obj.slug})
             work_el = etree.SubElement(el, "a", href=href)
+            if citation:
+                work_el.text = work_key
+            else:
+                work_el.text = work_obj.title
+                work_el.attrib["class"] = "italic"
```

Comment:
Deze gaat het niet doen, daar moet een CSS regel bij geschreven worden (Tailwind is uitgefaseerd). En als we dat doen, graag iets semantisch als `"work-title"` o.i.d.

Reply:
Ah okee, ik had het gekopieerd van die andere citation processor!